x-pack/elastic-agent/pkg/agent/application/info/agent_metadata.go

```diff
@@ -15,6 +15,57 @@ import (
 	"github.com/elastic/go-sysinfo/types"
 )
 
+// ECSMeta is a collection of agent related metadata in ECS compliant object form.
+type ECSMeta struct {
+	Agent *AgentECSMeta  `json:"agent"`
```

Comment:
@ph Same question as on the Kibana issue: I remember we said we prefix this agent meta data by `elastic.` as there might be other agent metadata which correlates better with the ECS defition.

Reply:
👍

Reply:
Yeah you are correct, forgot about that "elastic" specifif.

@ruflin Concerning that "elastic" I think we should make it a reserved key in the ECS specs?
Maybe @webmat has an opninion on that?

Reply:
The guidance is that no brand name (or proper names in general) should get into the schema. ECS should only contain concepts to which multiple technologies can map.

So there's no need to "reserve" the prefix `elastic.`, IMO.

Reply:
Okay, my concern was teams using namespaces and possible clash because in the "elastic" namespace. But I do understand the no "brand name" but I wonder if is something we should have internally or a process.